src/main/java/edu/hm/hafner/analysis/ast/FieldsAst.java

```diff
@@ -0,0 +1,70 @@
+package edu.hm.hafner.analysis.ast;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.puppycrawl.tools.checkstyle.api.DetailAST;
+import com.puppycrawl.tools.checkstyle.api.TokenTypes;
+
+/**
+ * Depicts the elements from the abstract syntax tree which are instance variables (i.e., fields).
+ *
+ * @author Christian Möstl
+ */
+public class FieldsAst extends Ast {
+    /**
+     * Creates a new instance of {@link FieldsAst}.
+     *
+     * @param fileName
+     *         the name of the Java file
+     * @param lineNumber
+     *         the line number that contains the warning
+     */
+    public FieldsAst(final String fileName, final int lineNumber) {
+        super(fileName, lineNumber);
+    }
+
+    private final List<DetailAST> instanceVariables = new ArrayList<>();
+
+    private void getInstanceVariables(final DetailAST element) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Linguistics Antipattern - The getter 'getInstanceVariables' should not return void linguistically](https://app.codacy.com/manual/uhafner/analysis-model/pullRequest?prid=5019963)